### DISCUSSION TITLE
is it directus support LSMCD cache ?

### DISCUSSION THREAD

**Discussion by iqraaaswad** [directus/directus#8891] [Archive]
is it directus support litespeed memcache ? could u give me some example about that pls?

**Comment by joselcvarela** [comment]
Hello @iqraaaswad 
~~I don't think so.
We rely on [keyv](https://github.com/jaredwray/keyv) for cache support. 
And since it seems to not have support for Litespeed, it is not possible for use to support it.~~

Edit, solution given by @rijkvanzanten :
> @joselcvarela that being said, Litespeed is designed as a memcached drop-in replacement. If Litespeed uses the same configuration and protocols as regular memcached, it should work as is 👍🏻

**Reply by rijkvanzanten** [reply]
@joselcvarela that being said, Litespeed is designed as a memcached drop-in replacement. If Litespeed uses the same configuration and protocols as regular memcached, it should work as is 👍🏻

**Reply by iqraaaswad** [reply]
ya.. i've try it myself.. and its work ^^
